### DISCUSSION TITLE
2019/04/menentukan-fenomena-dalam-penelitian

### DISCUSSION THREAD

**Discussion by giscus** [madi10/MANTANKODE#35] [General]
# 2019/04/menentukan-fenomena-dalam-penelitian

pentingnya melihat fenomena yang terjadi disekitar kita.

https://www.mantankode.net/2019/04/menentukan-fenomena-dalam-penelitian.html?m=1

**Comment by madi10** [comment]
Mengingat masa-masa Kuliah ❤️
